chainlibpy/grpc_client.py

```diff
@@ -44,28 +47,67 @@
 from chainlibpy.wallet import Wallet
 
 
+@dataclass
+class NetworkConfig:
+    grpc_endpoint: str
+    chain_id: str
+    address_prefix: str
+    coin_denom: str
+    coin_base_denom: str
+    derivation_path: str
+
+
+CRO_NETWORK = {
+    "mainnet": NetworkConfig(
+        grpc_endpoint="mainnet.crypto.org:9090",
+        chain_id="crypto-org-chain-mainnet-1",
+        address_prefix="cro",
+        coin_denom="cro",
+        coin_base_denom="basecro",
+        derivation_path="m/44'/394'/0'/0/0",
+    ),
+    "testnet_croeseid": NetworkConfig(
+        grpc_endpoint="testnet-croeseid-4.crypto.org:9090",
+        chain_id="testnet-croeseid-4",
+        address_prefix="tcro",
+        coin_denom="tcro",
+        coin_base_denom="basetcro",
+        derivation_path="m/44'/1'/0'/0/0",
+    ),
+}
+
+
 class GrpcClient:
     DEFAULT_GAS_LIMIT = 200000
 
     def __init__(
         self,
         wallet: Wallet,
-        chain_id: str,
-        grpc_endpoint: str,
+        network: NetworkConfig,
         credentials: ChannelCredentials = None,
     ) -> None:
```

Comment:
the interface changed, and the version needs to be updated too I think.

Reply:
Actually, the old version is not released yet.